words/words.md

```diff
@@ -358,117 +358,117 @@ But wait... haven't Taiwan and South Korea *already* contained COVID-19? For 4 w
     
-    **4 days on average to infecting someone else:** “The mean [serial] interval was 3.96 days (95% CI 3.53–4.39 days)” [Du Z, Xu X, Wu Y, Wang L, Cowling BJ, Ancel Meyers L](https://wwwnc.cdc.gov/eid/article/26/6/20-0357_article)
+	**4 jours en moyenne pour infecter quelqu'un d'autre :** « L'intervalle moyen [en série] était de 3,96 jours (IC 95% 3,53–4,39 jours) » [Du Z, Xu X, Wu Y, Wang L, Cowling BJ, Ancel Meyers L](https://wwwnc.cdc.gov/eid/article/26/6/20-0357_article)
     
-    **5 days on average to feeling symptoms:** “The median incubation period was estimated to be 5.1 days (95% CI, 4.5 to 5.8 days)” [Lauer SA, Grantz KH, Bi Q, et al](https://annals.org/AIM/FULLARTICLE/2762808/INCUBATION-PERIOD-CORONAVIRUS-DISEASE-2019-COVID-19-FROM-PUBLICLY-REPORTED)
+	**5 jours en moyenne pour ressentir des sypmtômes :** « La période d'incubation moyenne a été estimée à 5,1 jours (IC 95%, 4,5 à 5,8 jours) » [Lauer SA, Grantz KH, Bi Q, et al](https://annals.org/AIM/FULLARTICLE/2762808/INCUBATION-PERIOD-CORONAVIRUS-DISEASE-2019-COVID-19-FROM-PUBLICLY-REPORTED)
 
 ![](pics/timeline1.png)
 
-If cases only self-isolate when they know they're sick (that is, they feel symptoms), the virus can still spread:
+Si les cas s'auto-isolent seulement quand ils savent qu'ils sont malades (c'est-à-dire quand ils ressentent des symptômes), le virus peut encore se répandre :
 
 ![](pics/timeline2.png)
 
-And in fact, 44% of all transmissions are like this: *pre*-symptomatic! [^pre_symp]
+Et en fait, 44% de toutes les transmissions se font de cette façon: *pré*-symptomatique ! [^pre_symp]
 
-[^pre_symp]: “We estimated that 44% (95% confidence interval, 25–69%) of secondary cases were infected during the index cases’ presymptomatic stage” [He, X., Lau, E.H.Y., Wu, P. et al](https://www.nature.com/articles/s41591-020-0869-5)
+[^pre_symp]: « Nous avons estimé que 44% (intervall de confiance de 95%, 25–69%) des cas secondaires avaient été infectés pendant l'étape présymptomatique des cas index » [He, X., Lau, E.H.Y., Wu, P. et al](https://www.nature.com/articles/s41591-020-0869-5)
 
-But, if we find *and quarantine* a symptomatic case's recent close contacts... we stop the spread, by staying one step ahead!
+Mais, si nous trouvons *et mettons en quarantaine* les contacts récents d'un cas symptomatique… on arrête la propagation, en ayant toujours un coup d'avance !
 
 ![](pics/timeline3.png)
 
-This is called **contact tracing**. It's an old idea, was used at an unprecedented scale to contain Ebola[^ebola], and now it's core part of how Taiwan & South Korea are containing COVID-19!
+C'est ce qu'on appelle le **traçage des contacts**. C'est une vieille idée, elle a été utilisée à une échelle sans précédent pour contenir Ebola[^ebola], and maintenant elle est une partie centrale de comment Taiwan et la Corée du Sud contiennent le Covdi-19 !
```

Comment:
On avait convenu qu'utiliser "suivi" plutôt que "traçage", qui semble être le consesus

Reply:
and --> et
"de comment Taiwan et la Corée du Sud" --> "de la manière dont Taiwan et la Corée du Sud", plus propre selon moi

Reply:
Taiwan --> Taïwan